src/extension/devtools/devtools.ts

```diff
@@ -107,13 +97,6 @@ function startRequestInterval(ms = 500) {
   return () => clearTimeout(id);
 }
 
-const unsubscribers = new Set<() => void>();
-
-function unsubscribeFromAll() {
```

Comment:
When the devtools cancellation functions were much more complicated, I had a dream that they would all live here to made it easy to mass-cancel all of them. We never did end up doing much with that and we were able to simplify those other cancellation functions in the meantime.

This only now ever handles a single callback (the `cancelRequestInterval` function), so I have now named it as such to make this easier to follow.